src/main/java/com/notify/notify/model/SMS.java

```diff
@@ -0,0 +1,45 @@
+package com.notify.notify.model;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.sql.Timestamp;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+public class SMS {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @JsonProperty("id")
+    private Long id;
+
+    @JsonProperty("phone_number")
+    private String phoneNumber;
+
+    @JsonProperty("message")
+    private String message;
+    @JsonProperty("status")
+    private String status;
+
+    @JsonProperty("failure_code")
+    private String failureCode;
+
+    @JsonProperty("failure_comments")
+    private String failureComments;
+
+    @JsonProperty("created_at")
+    private Timestamp createdAt;
```

Comment:
please do not use import java.sql.Timestamp instead of this use LocalDateTime 
Read here:
https://www.geeksforgeeks.org/java-time-localdatetime-class-in-java/